test/tools/isoltest.cpp

```diff
@@ -361,8 +370,23 @@ Allowed options)",
 
 		po::notify(arguments);
 
+		if (arguments.count("no-ipc"))
+			disableIPC = true;
```

Comment:
Maybe output a warning if ``!ipcPath.empty()``? Maybe better not, though, I'm not sure.

Reply:
I think the idea was to at least compile all contracts defined in `test/libsolidity/semanticTest` if `no-ipc` was given. So we might want to warn about that.